modules/internal/ChapelRange.chpl

```diff
@@ -215,12 +215,21 @@ module ChapelRange {
     compilerError("Bounds of '..' must be integers of compatible types, when specified.");
   }
 
-  // Range builders for partially bounded ranges
-  proc chpl_build_partially_bounded_range(param bt: BoundedRangeType, bound: int(?w))
+  // Range builders for low bounded ranges
+  proc chpl_build_low_bounded_range(param bt: BoundedRangeType, bound: int(?w))
     return new range(int(w), bt, false, bound, bound);
-  proc chpl_build_partially_bounded_range(param bt: BoundedRangeType, bound: uint(?w))
+  proc chpl_build_low_bounded_range(param bt: BoundedRangeType, bound: uint(?w))
     return new range(uint(w), bt, false, bound, bound);
-  proc chpl_build_partially_bounded_range(param bt: BoundedRangeType, bound) {
+  proc chpl_build_low_bounded_range(param bt: BoundedRangeType, bound) {
+    compilerError("Bounds of '..' must be integers of compatible types, when specified.");
```

Comment:
Right now I have the following error messages. I'm not super thrilled with them, so let me know if there's something you like better

```chapel
  proc chpl_build_bounded_range(low, high) {
    compilerError("Bounds of 'low..high' must be integers of compatible types.");
  }

  proc chpl_build_low_bounded_range(low) {
    compilerError("Bounds of 'low..' must be an integer");
  }

  proc chpl_build_high_bounded_range(high) {
    compilerError("Bounds of '..high' must be an integer.");
}
```